recipes/glimmerhmm/build.sh

```diff
@@ -0,0 +1,59 @@
+if [ `uname` == Darwin ]; then
+    sed -i.bak "s~#include <malloc.h>~#include <malloc/malloc.h>~g" sources/util.c
+    sed -i.bak "s~#include <malloc.h>~#include <malloc/malloc.h>~g" sources/oc1.h
+    sed -i.bak "s~#include <malloc.h>~#include <malloc/malloc.h>~g" train/utils.c
+fi
+
+# fix typos in train makefile
+sed -i.bak "s|^escoreSTOP2:|scoreSTOP2:|g" train/makefile
+sed -i.bak "s|^rfapp:|erfapp:|g" train/makefile
+sed -i.bak "s| trainGlimmerHMM||g" train/makefile
+sed -i.bak "s|all:    build-icm|all:    misc.o build-icm.o build-icm-noframe.o build-icm|g" train/makefile
+
+# fix perl scripts
+sed -i.bak '1 s|^.*$|#!/usr/bin/env perl|g' train/trainGlimmerHMM
+sed -i.bak 's|FindBin;|FindBin qw($RealBin);|g' train/trainGlimmerHMM
+sed -i.bak 's|$FindBin::Bin;|"$RealBin/../share/glimmerhmm/train";|g' train/trainGlimmerHMM
+sed -i.bak '1 s|^.*$|#!/usr/bin/env perl|g' bin/glimmhmm.pl
+
+# make directories for storing training data and binaries
+mkdir -p $PREFIX/bin
+mkdir -p $PREFIX/share/glimmerhmm
+mkdir -p $PREFIX/share/glimmerhmm/train
+
+# cd into the sources
+if [ `uname` == Darwin ]; then
+        cd sources && make CC=$CXX
+        cd ../train && make clean && make all C=$CC CC=$CXX
+        cd ..
+else
+        cd sources && make CC=$GXX 
+        cd ../train && make clean && make all C=$CC CC=$CXX
+        cd ..
+fi
```

Comment:
Actually, this should be the same thing for osx and linux. Why the switch? CXX is set to the right thing on either architecture.